contracts/internal/balances/ExternalLending.sol

```diff
@@ -77,7 +73,10 @@ library ExternalLending {
         //
         // maxExternalUnderlyingLend = excessFunds / externalWithdrawThreshold
         uint256 maxExternalUnderlyingLend;
-        if (oracleData.currentExternalUnderlyingLend < oracleData.externalUnderlyingAvailableForWithdraw) {
+        if (
+            (oracleData.currentExternalUnderlyingLend < oracleData.externalUnderlyingAvailableForWithdraw) &&
+            (rebalancingTargetData.externalWithdrawThreshold != 0)
```

Comment:
If `externalWithdrawThreshold` is equal to zero that would imply an infinite maxExternalUnderlyingLend, I think. @T-Woodward